docs/partials/install-nrf91-sdk.md

```diff
@@ -18,18 +18,14 @@ Add the following to west.yml file in the manifest/projects subtree to add the G
   path: modules/lib/golioth
   revision: main
```

Comment:
We should consider changing that, so that this repo (docs) is describing steps to build with a specific version of SDK. Newer version (pointed by `main`) can have those steps a bit different over time.
```suggestion
  revision: 309597316d6963832bb777f901e5c869f99daff3
```